docs/en/guides/How-to-build.md

```diff
@@ -142,5 +142,24 @@ a oap image with name `bar/oap:foo`, run the following command
 ```
 5. Run `./mvnw compile -Dmaven.test.skip=true`
 6. Run `maven update`. Must remove the clean projects item before maven update(This will be clear the proto conversion Java file generated by the complie)
-7. Run `./mvnw compile` complie collector-remote-grpc-provider and apm-protocol
+7. Run `./mvnw compile` compile collector-remote-grpc-provider and apm-protocol
 8. Refresh project
+
+## Write necessary tests and run them
```

Comment:
Add this to here https://github.com/apache/skywalking/blob/master/docs/en/guides/README.md#for-code-developer . Make sense?

Reply:
Added

Reply:
I think we don't need this paragraph here. Build source doesn't mean write codes and contribute to upstream.

Reply:
I suggest move all to there

Reply:
Got it, I misunderstood your meaning, I thought you're asking me to put the link there, what a embarrass :-)